### PR TITLE
fixed dots in upper pagination

### PR DESCRIPTION
**Pull request by pablobuenaposada**
seems that someone forget to style the dots in the upper pagination bar

### CODE REVIEW

**Comment by tdonohue** [comment]
Thanks @pablobuenaposada !

Seems like a very minor fix we should just merge. But, I'll wait on a JSPUI user/committer to verify before doing so.

File: dspace-jspui/src/main/webapp/search/discovery.jsp [hunk 1]
```diff
@@ -486,7 +486,7 @@ else if( qResults != null)
 	
 	if (pageFirst != 1)
 	{
-	    %><li><a href="<%= firstURL %>">1</a></li><li>...</li><%
+	    %><li><a href="<%= firstURL %>">1</a></li><li class="disabled"><span>...<span></li><%
```
Comment (tdonohue) [inline dspace-jspui/src/main/webapp/search/discovery.jsp]: Just realized that this line should have a closing `span` tag.  So the code should be (`<span>...</span>`)

**Comment by pablobuenaposada** [comment]
Thanks @tdonohue, we were missing closing more than that span tag!

**Comment by kshepherd** [comment]
+1 tested on DSpace-Vagrant, JSPUI discovery with multi-page search results
<li class="disabled"><span>...</span></li>

**Comment by tdonohue** [comment]
This was merged in "master" and cherry-picked to "dspace-5_x" for the 5.3 release

**Pull request merged**